hudi-flink/src/main/java/org/apache/hudi/streamer/FlinkStreamerConfig.java

```diff
@@ -150,11 +145,12 @@
           description = "Whether to load partitions in state if partition path matching， default *")
   public String indexPartitionRegex = ".*";
 
-  @Parameter(names = {"--avro-schema-path"}, description = "Avro schema file path, the parsed schema is used for deserialization")
-  public String avroSchemaPath = "";
+  @Parameter(names = {"--avro-schema-path"}, description = "Avro schema file path, the parsed schema is used for deserialization",
+      required = true)
+  public String avroSchemaPath;
```

Comment:
Hmm. We should keep sync with `FlinkOptions`.

Reply:
@danny0405 thanks for the reply, let me take a look

Reply:
> `avro-schema-path` should not be required ? Because specify the `avroSchema` also works. They both should have a default value "".

@danny0405  I see the `READ_AVRO_SCHEMA_PATH` and `READ_AVRO_SCHEMA` in `FlinkOptions` have no default value. can we keep them the same?